tests/core/engine_adapter/docker/trino/catalog/datalake.properties

```diff
@@ -6,10 +6,6 @@ hive.s3.aws-access-key=minio
 hive.s3.aws-secret-key=minio123
 hive.metastore-cache-ttl=0s
 hive.metastore-refresh-interval=5s
-hive.metastore-timeout=10s
-hive.allow-drop-table=true
-hive.allow-add-column=true
-hive.allow-drop-column=true
-hive.allow-rename-column=true
-hive.allow-rename-table=true
+hive.metastore.thrift.client.connect-timeout=10s
```

Comment:
This just fixes some properties that were dropped / renamed between Trino 428 and Trino 452